apps/omg_eth/test/omg_eth/release_tasks/set_ethereum_client_test.exs

```diff
@@ -29,6 +29,8 @@ defmodule OMG.Eth.ReleaseTasks.SetEthereumClientTest do
         Enum.each(@configuration_old_ethereumex, fn {key, value} ->
           Application.put_env(:ethereumex, key, value, persistent: true)
         end)
+
```

Comment:
Wow, we test release tasks - that's great

Reply:
Amazing 🥇